core/storage/topic/gae_models.py

```diff
@@ -388,6 +383,14 @@ class TopicSummaryModel(base_models.BaseModel):
     # The thumbnail background color of the topic.
     thumbnail_bg_color = datastore_services.StringProperty(indexed=True)
     version = datastore_services.IntegerProperty(required=True)
+    # A mapping whose key is a story id of a published story that belongs to
```

Comment:
A dict that maps published story IDs to lists of exploration IDs. Each list of exploration IDs represents ...

(Rationale: We typically refer to dicts in the python codebase. Also when we say the "mapping whose key" it suggests that there is only one key in the dict.)

Reply:
Done.